specification/servicebus/data-plane/servicebus-swagger.json

```diff
@@ -766,6 +766,14 @@
             "name": "ForwardDeadLetteredMessagesTo",
             "namespace": "http://schemas.microsoft.com/netservices/2010/10/servicebus/connect"
           }
+        },
+        "maxMessageSizeInKilobytes": {
+          "description": "The maximum message payload that can be accepted by the queue.",
```

Comment:
This description does not mention anything about the size...

Reply:
my bad, I copied the phrase from portal but didn't check the wording.

I've added several words, how does it look to you now @ramya-rao-a

Reply:
looks good, thanks!